.gitignore

```diff
@@ -6,6 +6,7 @@
 vendor/
 composer.phar
 test/integration/test/integration/wovn_index_sample_workspace
+.phpunit/
```

Comment:
Is this needed?

Reply:
Coverage report is now outputted to .phpunit. If I create coverage report locally, we should not commit it by mistake.

Reply:
Ah, I think this commit is OK.
It is useful to develop locally.